storio/src/main/java/com/pushtorefresh/storio/ILogger.java

```diff
@@ -0,0 +1,19 @@
+package com.pushtorefresh.storio;
+
+public interface ILogger {
+    void v(String tag, String message);
```

Comment:
Also, I think we can left only messages of different levels, without tags, tag can be set as final property in Logger impl constructor, it's not logger for application, just small library :)